### PR TITLE
Add API for Edge

### PR DESCRIPTION
**Pull request by tikhsuP**
This PR adds API for Microsoft Edge Browser.

### CODE REVIEW

**Comment by cgorringe** [comment]
It looks fine. I think I'll also push a test for Safari to this PR in a bit.

**Comment by cgorringe** [comment]
Updated to detect most major browsers.

File: webextension/scripts/utils.js [hunk 1]
```diff
@@ -23,6 +19,50 @@ const newshosts = new Set([
   'www.washingtonpost.com'
 ])
 
+/* * * Browser Detection * * */
+
+function getBrowser() {
+  // the order of these is important!
+  if (navigator.brave) { return 'brave' }
+  else if (navigator.userAgent.indexOf('Edg') !== -1) { return 'edge' }
```
Comment (tikhsuP) [inline webextension/scripts/utils.js:27]: Should we detect `indexOf( )` for both 'Edge' (Legacy Version) & 'Edg' (Chromium verison)?

Comment (cgorringe) [reply webextension/scripts/utils.js:27]: Yes, you're correct.

File: webextension/scripts/utils.js [hunk 2]
```diff
@@ -23,6 +19,50 @@ const newshosts = new Set([
...
+
+const feedbackURLs = {
+  chrome: 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak/reviews?hl=en',
+  chromium: 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak/reviews?hl=en',
+  firefox: 'https://addons.mozilla.org/en-US/firefox/addon/wayback-machine_new/',
+  safari: 'https://apps.apple.com/us/app/wayback-machine/id1201888313'
+}
+
+const gBrowser = getBrowser()
+const isChrome = (gBrowser === 'chrome') || (gBrowser === 'chromium')
```
Comment (anishsarangi) [inline webextension/scripts/utils.js:56]: Do we really need these variables (except isFirefox) at this point of time ?

Comment (cgorringe) [reply webextension/scripts/utils.js:56]: I'm going to need them in the future, possibly. I'll have to make some changes like converting all `chrome` to `browser` and there are some browser-specific issues, so to get it all working with Safari as well. So let's keep it for now.

Comment (anishsarangi) [reply webextension/scripts/utils.js:56]: Ok. Thank you. !

**Pull request merged**